persistence/sql/persister_session.go

```diff
@@ -128,12 +135,23 @@ func (p *Persister) ListSessionsByIdentity(ctx context.Context, iID uuid.UUID, a
 	nid := p.NetworkID(ctx)
 
 	if err := p.Transaction(ctx, func(ctx context.Context, c *pop.Connection) error {
+		i, err := p.GetIdentity(ctx, iID)
+		if err != nil {
+			return sqlcon.HandleError(err)
+		}
+
 		q := c.Where("identity_id = ? AND nid = ?", iID, nid)
 		if except != uuid.Nil {
 			q = q.Where("id != ?", except)
 		}
 		if active != nil {
 			q = q.Where("active = ?", *active)
+
+			if *active {
+				q.Where("expires_at >= ?", time.Now())
+			} else {
+				q.Where("expires_at < ?", time.Now())
```

Comment:
At some point we realized that all DB timestamps need to be UTC

```suggestion
				q.Where("expires_at >= ?", time.Now().UTC())
			} else {
				q.Where("expires_at < ?", time.Now().UTC())
```